core/src/main/java/com/exadel/aem/toolkit/core/lists/utils/ListHelper.java

```diff
@@ -90,6 +97,69 @@ private ListHelper() {
             .collect(Collectors.toList());
     }
 
+    /**
+     * Creates a list of entries under given {@code path} based on list of {@link SimpleListItem}
+     * @param resourceResolver Sling {@link ResourceResolver} instance used to create the list
+     * @param path             JCR path of the items list
+     * @param listItems        List of {@link SimpleListItem}
+     * @return Created page containing list of entries or ${@code null} if page cannot be created
```

Comment:
Is `${` this a typo or this should be here? because in java doc it looks like this:
![image](https://user-images.githubusercontent.com/62335469/145381519-cc32bfd6-a05c-4b4e-8812-bd157dc116b5.png)